tests/integration/test_utils/data_source_config/snowflake.py

```diff
@@ -38,11 +33,15 @@ def pytest_mark(self) -> pytest.MarkDecorator:
 
     @override
     def create_batch_setup(
-        self, data: pd.DataFrame, request: pytest.FixtureRequest
+        self,
```

Comment:
The changes here should look very similar to in `postgres.py`. We'll DRY it up in a subsequent PR.